src/main/java/com/ning/http/util/UriEncoder.java

```diff
@@ -92,6 +92,7 @@ protected String withQueryWithParams(final String query, final List<Param> query
             // concatenate raw query + raw query params
             StringBuilder sb = StringUtils.stringBuilder();
             sb.append(query);
+            b.append('&');
```

Comment:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:compile (default-compile) on project grizzly-http-client: Compilation failure
[ERROR] src/main/java/com/ning/http/util/UriEncoder.java:[95,13] cannot find symbol
[ERROR]   symbol: variable b
```

If you decide to continue this PR - please rebase against current default branch.